wordpress-mobile/WordPress-Android
# RI: Propagate selected "chip" state into the VM, introduce "Done" button

Master issue: #12027

This PR includes following tasks:
- [ ] Make the "chips" selectable and propagate selected items into the VM
- [ ] Introduce "Done" button - navigate to Discover and pass the selected topics to ReaderRepository

### To test:
- Go to Reader tab
- Notice that Done button is initially hidden
- Wait for the interests "chips" to load
- Notice that Done button is disabled with text "Select a few to continue"
- Select few interest chips
- Notice that Done button is enabled with text "Done"
- De-select all interest chips
- Notice that Done button is disabled with text "Select a few to continue"
- Select an interest chip to enable Done button and click it
- Existing Reader is shown with the Discover tab selected (yet to replace Discover tab with the new blank Discover screen)


![done](https://user-images.githubusercontent.com/1405144/83748913-8542e580-a680-11ea-9a63-776acffc95f0.gif)

Note:

Need to refine
1. styling
2. transition to Discover screen
will do in upcoming PRs.

PR submission checklist:
